crates/core/src/surfnet/locker.rs

```diff
@@ -731,7 +737,7 @@ impl SurfnetSvmLocker {
     }
 
     pub fn do_profile_instructions(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_instruction_profiling_enabled(&self) -> bool {
```
could be more readable / conventional

Reply:
Done